docs/detections/value-list-exceptions.asciidoc

```diff
@@ -37,27 +37,54 @@ act as delimiters.
 =========================
 
 . Go to *Rules* -> *Detection rules (SIEM)*.
-. Click *Import value lists*. The *Import value lists* window opens.
+. Click *Manage value lists*. The *Manage value lists* window opens.
 +
 [role="screenshot"]
-image::images/upload-lists-ui.png[Import value lists flyout,75%]
+image::images/upload-lists-ui.png[Manage value lists flyout,75%]
 
 . Select the list type (*Keywords*, *IP addresses*, *IP ranges*, or *Text*) from the *Type of value list* drop-down.
 . Drag or select the `csv` or `txt` file that contains the values.
 . Click *Import value list*.
 
-NOTE: When the name of the file you are importing already exists, the values in
-the new file are appended to the previously imported values.
+NOTE: If you import a file with a name that already exists, a new list is not created. The imported values are added to the existing list instead.
 
-[[edit-value-lists]]
+[[manage-value-lists]]
 [discrete]
 === Manage value lists
 
-To view, delete, or export existing value lists:
+You can edit, remove, or export existing value lists. 
```

Comment:
Update all of the icons on this page.

Reply:
Resolved in [735dff1](https://github.com/elastic/security-docs/pull/5195/commits/735dff16af04cc810bbb71283cb0dbd65054115e).